### DISCUSSION TITLE
Get the name of current applied theme from terminal

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1384] [Q&A] [answered]
I am trying to make a script that depends on the name of current applied theme. Is there a way to know that perhaps if hyde exposes a global variable?

**Comment by dieBakterie** [comment]
In the `theme.conf` is the actual Theme or in the `hyde.conf` just `${theme} < cat .theme/theme.conf`

**Reply by rubiin** [reply]
where is the location of theme.conf . Its not on home

**Comment by kRHYME7** [comment, marked as answer]
@rubiin  if you source the globalcontrol.sh will fetch the current hydeTheme for yah

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/80330ad9-c09a-44ea-9704-ebf50ae507d2)

**Reply by rubiin** [reply]
cool
